Header header
poses[] enemylists
poses[] friendlists